app/controllers/jobs-controller.js

```diff
@@ -37,7 +37,8 @@ router.post('/new', (req, res, next) => {
       new AddJobViewModel(accountId, req.body, req.validationErrors()));
   }
 
-  const jobData = Object.assign({}, req.body, { accountId, status: progression[0] },
+  const jobData = Object.assign({}, req.body,
+    { accountId, status: progression[0], status_sort_index: 0 },
     { deadline: parseDeadline(deadline) });
```

Comment:
Might be simpler to have only one object that gets merged with the request body?

```javascript
const jobData = Object.assign({}, req.body, {
  accountId,
  deadline: parseDeadline(deadline),
  status: progression[0],
  status_sort_index: 0,
});
```